repository/copier.go

```diff
@@ -111,6 +111,22 @@ func (c *HDFSCopier) CopyFromRemote(src, dst string, localFs billy.Filesystem) (
 	return
 }
 
+func (c *HDFSCopier) deleteIfExists(file string) error {
+	_, err := c.client.Stat(file)
+	if err == nil {
+		err = c.client.Remove(file)
+		if err != nil {
+			return err
+		}
+	}
+
+	if err != nil && !os.IsNotExist(err) {
```

Comment:
if you move this up the `err == nil` you can simplify this method to just:

```go
func (c *HDFSCopier) deleteIfExists(file string) error {
        if _, err := c.client.Stat(file); os.IsNotExist(err) {
                return nil
        } else if err != nil {
                return err
        }

        return c.client.Remove(file)
}
```

Reply:
Nice one! I didn't know about the fancy if statement. :sweat_smile: